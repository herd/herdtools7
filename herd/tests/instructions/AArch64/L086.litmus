AArch64 L086
{
0:X0=instr:"ADR X1,.+0";
}
  P0          ;
L0:           ;
  ADR X1,L0   ;
  LDR W2,[X1] ;
  CMP W0,W2   ;
  CSET W3,EQ  ;
forall 0:X3=1  