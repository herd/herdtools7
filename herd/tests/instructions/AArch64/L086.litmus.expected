Test L086 Required
States 1
0:X3=1;
Ok
Witnesses
Positive: 1 Negative: 0
Condition forall (0:X3=1)
Observation L086 Always 1 0
Hash=5e2daad1656881d22f1562e268ebe492

